galaxy_ng/app/management/commands/set-repo-keyring.py

```diff
@@ -13,7 +13,7 @@
 from pulpcore.plugin.tasking import dispatch
 
 # Set logging_uid, this does not seem to get generated when task called via management command
-django_guid.set_guid(django_guid.utils.generate_guid)
+django_guid.set_guid(django_guid.utils.generate_guid())
```

Comment:
cool, this matches how it is called in pulp https://github.com/pulp/pulpcore/blob/main/pulpcore/tasking/tasks.py#L120